packages/devtools_app/lib/src/screens/memory/panes/leaks/diagnostics/leak_analyzer.dart

```diff
@@ -10,8 +10,8 @@ import '../../../shared/heap/spanning_tree.dart';
 import 'model.dart';
 
 /// Analyzes notGCed leaks and returns result of the analysis.
-NotGCedAnalyzed analyseNotGCed(NotGCedAnalyzerTask task) {
-  analyzeHeapAndSetRetainingPaths(task.heap, task.reports);
+Future<NotGCedAnalyzed> analyseNotGCed(NotGCedAnalyzerTask task) async {
```

Comment:
Ubernit: use analyze instead of analyse for consistency.

Reply:
resolved in https://github.com/flutter/devtools/pull/5134